### ISSUE TITLE
Redesign article subscription interface

### ISSUE THREAD

**Issue by tylersticka** [cloudfour/cloudfour.com-patterns#1339]
While reviewing @gerardo-rodriguez's work on #1094, we ran into some interesting challenges relating to the interaction between visual control labels and audible control labels. This was summarized nicely by @gerardo-rodriguez here: https://github.com/cloudfour/cloudfour.com-patterns/pull/1318#issuecomment-867210632

We realized that there may be an opportunity to redesign that area in order to accomplish two main goals:

- Make the action to be taken… enabling or disabling notifications… clear to users of all devices without relying on wholly separate labels. This will be in keeping with our commitment to both responsiveness and accessibility.
- Perhaps make the interface element as a whole easier to use in both the current article context and in the article listing, which currently _only_ supports email notifications.

As part of a voice conversation (which also included @dromo77, @AriannaChau and @haysjoey), @gerardo-rodriguez shared a link to an example notification UI from [Inclusive Components](https://inclusive-components.design/toggle-button/):

<img width="848" alt="Screen Shot 2021-06-24 at 3 45 45 PM" src="https://user-images.githubusercontent.com/69633/123341698-4558d580-d503-11eb-9380-f84f56a8e4be.png">

Other ideas included more explanatory copy beforehand.

Outcomes of this issue should be a prototype or wireframes reviewed by me for creative direction and @gerardo-rodriguez for accessibility.

**Comment by gerardo-rodriguez** [comment]
I haven't taken the time to verify the UX, but I noticed Github has a notifications button. Perhaps we can learn something from how they handle it?

<img width="706" alt="Screen Shot 2021-07-21 at 10 53 56 AM" src="https://user-images.githubusercontent.com/459757/126536663-eb5b2681-4df9-4c48-aa54-a7aca91e3482.png">
